wagtail-{{ cookiecutter.project_name_kebab }}/.github/publish.yml

```diff
@@ -0,0 +1,36 @@
+# See https://packaging.python.org/en/latest/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/
+# for a detailed guide
+name: Publish to PyPI
+
+on:
+  release:
+    types: [released, prereleased]
+
+permissions:
+  contents: read # to fetch code (actions/checkout)
```

Comment:
The default for new GitHub repos is now that every workflow gets `contents: read` and you have to scope up to get write (it used to be the other way round - workflows got write by default, you scope down to read-only). Given the default for a new repo is now


![166982234-6d2e56ca-d5dd-4028-b04d-3e701010c269](https://user-images.githubusercontent.com/6025893/205280872-68fbd06f-d6e4-4aee-8f74-57d53fd84f05.png)

and this is a project specifically for generating new repos, I reckon we could just omit this as we're re-stating the default.

Reply:
I would rather be explicit, than implicit. Especially for any packages that move to using the cookiecutter template, rather than start with it. e.g. https://github.com/wagtail-nest/wagtail-font-awesome-svg/pull/4